microsoft-edge/devtools-guide-chromium/memory-problems/heap-snapshot-schema.md

```diff
@@ -303,8 +303,72 @@ The first number in the group of numbers for an edge in the `edges` array corres
 | Shortcut | An easier-to-read representation of some other path. This type is rarely used. For example, if you use `Function.prototype.bind` to create a bound function with some bound arguments, V8 creates a `JSBoundFunction`, which points to a `FixedArray` (an internal type), which points to each bound argument. When producing a snapshot, V8 adds a shortcut edge from the bound function directly to each bound argument, bypassing the `FixedArray`. |
 | Element | Object properties where the key is a number. |
 
+#### locations
+
+The `locations` array, which is at the top level of the `.heapsnapshot` data, contains information about the location of `HeapSnapshot` nodes.
```

Comment:
```suggestion
The `locations` array, which is at the top level of the `.heapsnapshot` data, contains information about where each node in the snapshot was created.
```

Feel free to change if this is incorrect. This is based on the assumption that location, here, really means where was a node created/instantiated in the JavaScript codebase.

Reply:
the present thread is Outdated, which means the surrounding file is outdated here.  Latest view:
https://github.com/MicrosoftDocs/edge-developer/pull/2519/files

Reply:
it changed to:
The `locations` array, which is at the top level of the `.heapsnapshot` data, contains information about where some of the nodes in the snapshot were created.

gh rendered:
https://github.com/christyycchen/edge-developer/blob/patch-1/microsoft-edge/devtools-guide-chromium/memory-problems/heap-snapshot-schema.md#locations

Reply:
@christyycchen @captainbrosset This comment can be Resolved.